sleap_roots_training/config.yaml

```
@@ -1,6 +1,6 @@
-collection_name: null
-entity_name: eberrigan-salk-institute-for-biological-studies
-experiment_name: 20250530_younger_rice_seminal_sweep_input_scaling
-job_type: null
-project_name: sleap-roots
-registry: sleap-roots-models
+collection_name: 20250625_cyl_arabidopsis_primary_receptive_field
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)